src/com/vinnik/kate/Main.java

```diff
@@ -0,0 +1,21 @@
+package com.vinnik.kate;
+
+/** Class, which implements interaction with user. */
+public class Main {
+    private final static int MINIMAL_AMOUNT_OF_THREADS = 0;
+    public static void main(String[] args) {
+        try {
+            final int amountOfThreads = Integer.parseInt(args[0]);
+            if (amountOfThreads < MINIMAL_AMOUNT_OF_THREADS) {
+                System.out.println("Wrong number format. Please, enter positive integer number. ");
+            }
+            MultithreadingSorter.measureTimeOfSorting(amountOfThreads);
+        } catch (java.lang.ArrayIndexOutOfBoundsException e) {
+            System.out.println("Please, enter amount of threads.");
+        } catch (java.lang.NumberFormatException e) {
+            System.out.println("Wrong number format. Please, enter positive integer number. ");
+        } catch (Exception e) {
```

Comment:
I think it makes sense to print on the usage of your program. In case a user would enter the invalid parameters, she/he would see the correct usage immediately.